packages/react-vis/package.json

```diff
@@ -86,6 +103,7 @@
     "uglify-js": "^2.8.22"
   },
   "peerDependencies": {
+    "@babel/core": "^7.0.0",
```

Comment:
Hm why do we need this peer dependency? Aren't we compiling the bundle?

We may want to follow the practices on baseweb if we need a reference configuration.

Reply:
I agree about using baseweb as a reference, the purpose of both projects are similar (components library) and they managed to keep it simple enough

Reply:
oops... I'll remove this.